indra/llwebrtc/llwebrtc_impl.h

```diff
@@ -35,6 +35,7 @@
 #define WEBRTC_POSIX 1
 #elif __linux__
```

Comment:
I suppose the LL-way is to check for LL_LINUX, LL_WINDOWS, LL_DARWIN. This makes grepping easier.

Which at the same time makes me curious why there is a need for WEBRTC_xxx when there is already such a thing as LL_xxx.

And another small nitpick
LL_**WINDOWS** <> LL_**WIN**
LL_**DARWIN** <> WEBRTC_**MAC** (you're my hero btw if you'd manage to get DARWIN changed to OSX or MAC)

Reply:
I'm trying to use the WebRTC way of doing everything (thread model, memory, platform identifiers, etc.) in llwebrtc.  It's the 'interface' that wraps webrtc in a very simple wrapper for the viewer.  Ultimately, the goal is to move it out of the viewer into something downloaded by autobuild (or whatever a TPV dev uses) where I won't have access to viewer-related headers and build-system constants.

Reply:
Got it, that makes sense.
For grepability  I'd still prefer _WINDOWS and _DARWIN (even though I'd personally prefer _MAC)
But then  the suffix is the same across the sources.

Reply:
The constants listed are used in the webrtc headers so I'm kinda stuck using WEBRTC_*, unfortunately.